recipes-debian/iptables/iptables-persistent_debian.bb

```diff
@@ -0,0 +1,24 @@
+SUMMARY = "boot-time loader for netfilter configuration"
+LICENSE = "GPLv3"
+LIC_FILES_CHKSUM = "file://debian/copyright;md5=efa38437dfaf00cb7e3dd14dc520ca65"
+
+inherit debian-package
+require recipes-debian/sources/iptables-persistent.inc
+DEBIAN_PATCH_TYPE="quilt"
+DEBIAN_QUILT_PATCHES = ""
+
+RDEPENDS_${PN} = "iptables"
+
+do_install() {
+    oe_runmake 'DESTDIR=${D}' install
+
+    install ${S}/plugins/* ${D}/usr/share/netfilter-persistent/plugins.d/.
```

Comment:
You can use ${datadir} as /usr/share. such variables are defined in [bitbake.conf](http://git.yoctoproject.org/cgit.cgi/poky/plain/meta/conf/bitbake.conf?h=warrior)

Reply:
Thanks. Updated.

Reply:
LGTM :)